### PR TITLE
Added cursor type to nav buttons

### PR DESCRIPTION
**Pull request by Belicosus**
Cursor wasn't showing correctly on li elements in .nav class.
cursor now set to type "pointer".

### CODE REVIEW

**Comment by Nopzen** [comment]
:+1: this :) would be my first addition aswell!

**Comment by dimitrigogios** [comment]
Cursor is not showing correctly because there is no href on the a. That's the mistake. I'll leave the scss-file as it for now :)

File: origin/scss/components/navs/_navs.scss [hunk 1]
```diff
@@ -3,9 +3,12 @@
 	padding: 0;
 
 	li {
+		cursor: pointer;
```
Comment (dimitrigogios) [inline origin/scss/components/navs/_navs.scss]: Actually, I've made a helperclass called .pointer. Instead of this I would prefer: @extend .pointer

**Comment by Belicosus** [comment]
Okay, two comments:
You should add this helper pointer class to any navigation items. I can't imagine a scenario where you don't wont the cursor to be the pointer.

Second, I think it would be more intuitive if the href tag wasn't mandatory. It's getting more and more common to have some JS handle state changes.

**Comment by dimitrigogios** [comment]
Yes, I completely agree with your points. You're are welcome to make a new pull-request with the added helperclass. If you want? :)

**Pull request closed without merging**